sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ViewTest.java

```diff
@@ -1378,51 +1369,51 @@ private void testViewNonmerging(
 
   @Test
   public void testViewUnboundedAsSingletonDirect() {
-    testViewUnbounded(createTestDirectRunner(), View.<KV<String, Integer>>asSingleton());
+    testViewUnbounded(TestPipeline.create(), View.<KV<String, Integer>>asSingleton());
```

Comment:
I don't believe that's the case - a Globally Windowed Unbounded `PCollection` with default triggering should always fail to create a `PCollectionView` at Pipeline construction time. Given the current formulation of `CreatePCollectionView`, that seems like a reasonable test.